src/parser/jobs/rpr/modules/OtherGauges.tsx

```diff
@@ -0,0 +1,142 @@
+		[this.data.actions.SOUL_SCYTHE.id, {action: 50}],
+		// Spenders
+		[this.data.actions.BLOOD_STALK.id, {action: -50}],
+		[this.data.actions.GRIM_SWATHE.id, {action: -50}],
+		[this.data.actions.UNVEILED_GIBBET.id, {action: -50}],
+		[this.data.actions.UNVEILED_GALLOWS.id, {action: -50}],
+		[this.data.actions.GLUTTONY.id, {action: -50}],
+	])
+
+	override initialise() {
+		super.initialise()
+
+		const playerFilter = filter<Event>().source(this.parser.actor.id)
+		const foeFilter = filter<Event>().actor(oneOf(this.actors.foes.map(a => a.id)))
+
+		// Soul Gauge
+		const soulActions = Array.from(this.soulGaugeModifiers.keys())
+		this.addEventHook(playerFilter.type(oneOf(['action', 'combo'])).action(oneOf(soulActions)), this.onSoulModifier)
+		this.addEventHook(foeFilter.type('death'), this.onFoeDeath)
+
+		// Shroud Gauge
+		this.addEventHook(playerFilter.type('action').action(this.data.matchActionId(SHROUD_ACTIONS)), this.onShroudModifier)
+
+		this.addEventHook('complete', this.onComplete)
+	}
+
+	private onFoeDeath(event: Events['death']) {
+		// We effectively check if they had the status 100ms before death, since it drops with death
+		if (this.actors.get(event.actor).at(event.timestamp - 100).hasStatus(this.data.statuses.DEATHS_DESIGN.id)) {
+			this.soulGauge.generate(DESIGN_GAIN)
+		}
+	}
+
+	private onSoulModifier(event: Events['action' | 'combo']) {
+		const modifier = this.soulGaugeModifiers.get(event.action)
+
+		if (modifier != null) {
+			const amount = modifier[event.type] ?? 0
+			this.soulGauge.modify(amount)
+		}
+	}
+
+	private onShroudModifier(event: Events['action']) {
+		const action = this.data.getAction(event.action)
+		if (action == null) { return }
+
+		switch (action.id) {
+		case this.data.actions.ENSHROUD.id:
+			this.shroudGauge.spend(HIGH_SHROUD_MOD)
+			break
+
+		case this.data.actions.PLENTIFUL_HARVEST.id:
+			this.shroudGauge.generate(HIGH_SHROUD_MOD)
+			break
+
+		default:
+			this.shroudGauge.generate(BASE_SHROUD_MOD)
+		}
+	}
+
+	private onComplete() {
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.SOUL_SLICE.icon,
+			content: <Trans id="rpr.gauge.soul.suggestions.overcap.content">
+					Avoid letting your Soul gauge overcap. The wasted resources cost you Shroud generation which you need for your highest damage skills.
+			</Trans>,
+			why: <Trans id="rpr.gauge.soul.suggestions.overcap.why">
+				{this.soulGauge.overCap} Souls lost to overcapping Soul gauge.
+			</Trans>,
+			tiers: SUGGESTION_TIERS,
+			value: this.soulGauge.overCap,
+		}))
+
+		this.suggestions.add(new TieredSuggestion({
+			icon: this.data.actions.PLENTIFUL_HARVEST.icon,
+			content: <Trans id="rpr.gauge.shroud.suggestions.overcap.content">
+					Avoid letting your Shroud gauge overcap as it can cost you uses of <DataLink action="ENSHROUD"/> over the fight.
+			</Trans>,
+			why: <Trans id="rpr.gauge.shroud.suggestions.overcap.why">
+				{this.soulGauge.overCap} Shrouds lost to overcapping Shroud gauge.
```

Comment:
problem: you're using the soul gauge value for shroud gauge
nit: your soul gauge overcap should really be `⌈this.shroudGauge.overCap / 50⌉` or the text should be changed to reflect that it's shroud gauge lost, instead of being full uses of enshroud.
```suggestion
				{this.shroudGauge.overCap} Shrouds lost to overcapping Shroud gauge.
```

Reply:
Good catch. Not sure about showing actual Enshrouds, the game itself calls each point of the gauge a shroud so it's kinda awkward naming here. Like Soul gauge, I'm open to changing this, just wanna see what users prefer.